pkg/apis/internal.acorn.io/v1/appinstance.go

```diff
@@ -73,6 +76,7 @@ type AppInstanceSpec struct {
 	AutoUpgrade         *bool            `json:"autoUpgrade,omitempty"`
 	NotifyUpgrade       *bool            `json:"notifyUpgrade,omitempty"`
 	AutoUpgradeInterval string           `json:"autoUpgradeInterval,omitempty"`
+	WorkloadClass       WorkloadClass    `json:"workloadClass,omitempty"`
 	Memory              Memory           `json:"memory,omitempty"`
```

Comment:
Is Memory no longer used?

Reply:
Memory is still used in the case that there is no WorkloadClass. It will also be used in combination with a WorkloadClass. For example:

```sh
$ acorn run --memory 512Mi --workload-class=extra-large .
```